kapitan/utils.py

```diff
@@ -552,6 +552,11 @@ def unpack_downloaded_file(file_path, output_path, content_type):
             tar.extractall(path=output_path)
             tar.close()
             is_unpacked = True
+        elif (r"\.zip$", file_path):
```

Comment:
did you mean 

```
elif re.search(r"\.zip$", file_path):
```

Reply:
> did you mean
> 
> ```
> elif re.search(r"\.zip$", file_path):
> ```

ohhh, that's was a stupid  typo from my side, and I got false-positive tests..
I spent time to investigate a problem and completely changed PR.
Please take a look.